### PR TITLE
pages.{en, pt_BR}: replace the previous dockerd URL with the new one

### PR DESCRIPTION
**Pull request by rilysh**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

Edit: There was another page (`./pages.ar/linux/dockerd.md`) that has the old URL.. but it's in Arabic and I can't seem to able to edit it.

### CODE REVIEW

**Comment by vitorhcl** [comment]
Could you please use `set-more-info-libk.py` to upsate the other link?

Its usage is documented in the script itself.

**Review by isaacvicente — APPROVED** [review]
LGTM :+1:

File: pages.pt_BR/linux/dockerd.md
[the remark this answers is not in the document]
Comment (rilysh) [reply pages.pt_BR/linux/dockerd.md:1]: thanks, done

**Review by vitorhcl — APPROVED** [review]

**Comment by vitorhcl** [comment]
Thank you!

**Pull request merged**